src/api/core/accounts.rs

```diff
@@ -352,10 +364,28 @@ async fn post_kdf(data: JsonUpcase<ChangeKdfData>, headers: Headers, mut conn: D
         err!("Invalid password")
     }
 
-    if data.KdfIterations < 100_000 {
-        err!("KDF iterations lower then 100000 are not allowed.")
+    if data.Kdf == UserKdfType::Pbkdf2 as i32 && data.KdfIterations < 100_000 {
+        err!("PBKDF2 KDF iterations lower than 100000 are not allowed.")
```

Comment:
```suggestion
        err!("PBKDF2 KDF iterations must be at least 100000.")
```
(to make the wording more consistent with the Argon2 case)